conda_smithy/configure_feedstock.py

```diff
@@ -31,6 +31,7 @@
     remove_file_or_dir,
 )
 from conda_smithy.utils import get_feedstock_name_from_meta
+from conda_smithy import azure_ci_utils
```

Comment:
Importing this here triggers a No azure token found warning.

Reply:
We could do a better job of this, but yeah, putting these imports right when they are needed is important.